docs/userguide/format_parse/messages/try-mf2.md

```diff
@@ -0,0 +1,366 @@
+---
+layout: default
+title: Trying MF 2.0 Final Candidate
+nav_order: 3
+parent: Formatting Messages
+grand_parent: Formatting
+---
+<!--
+© 2023 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Trying MF 2.0 Final Candidate
+{: .no_toc }
+
+## Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+# Hello `MessageFormat2`
+
+
+
+
+## C++ Linux & macOS
+
+
+1. Prepare a sandbox folder
+    ```sh
+    export ICU_SANDBOX=~/hello_icu_mf2
+    mkdir $ICU_SANDBOX
+    cd $ICU_SANDBOX
+    ```
+
+1. Build ICU4C (you only need to do this once)
+
+    ```sh
+    git clone https://github.com/unicode-org/icu.git
+    pushd icu/icu4c/source
+
+    # Run this and choose the platform and toolchain you prefer
+    ./runConfigureICU --help
+
+    # if macOS
+    ./runConfigureICU macOS/gcc
+    # else if Linux  (gcc is just an example, there are 5 Linux options)
+    ./runConfigureICU Linux/gcc
+    # end
+
+    export DESTDIR=$ICU_SANDBOX/icu_release
+    make -j8 releaseDist
+    popd
+    ```
+
+1. Create a minimal C++ file here (we are in the `$ICU_SANDBOX` folder).
+Let's call it `hello_mf2.cpp`
+    ```cpp
+    // hello_mf2.cpp
+    #include <iostream>
+
+    #include "unicode/utypes.h"
+    #include "unicode/calendar.h"
+    #include "unicode/errorcode.h"
+    #include "unicode/locid.h"
+    #include "unicode/messageformat2.h"
+
+    using namespace icu;
+
+    int main() {
+       ErrorCode errorCode;
+       UParseError parseError;
+
+       icu::Calendar* cal(Calendar::createInstance(errorCode));
+       cal->set(2025, Calendar::JANUARY, 28);
```

Comment:
The text on line 112 says the message will say "January 13", but this sets the calendar to the 28th???

Reply:
Thanks, updated (created suggestion in my review)